### PR TITLE
NEW: add mechanism to skip test in case of Out of Quota Error

### PR DESCRIPTION
**Pull request by asanzgom**
closes [MGDSTRM-10432](https://issues.redhat.com/browse/MGDSTRM-10432)

### CODE REVIEW

**Review by Frawless — APPROVED** [review]

File: lib/kafka.ts [hunk 1]
```diff
@@ -18,6 +18,16 @@ export const createKafkaInstance = async function (
   // FIXME: workaround for https://github.com/redhat-developer/app-services-ui-components/issues/590
   // https://github.com/microsoft/playwright/issues/15734#issuecomment-1188245775
   await new Promise((resolve) => setTimeout(resolve, 500));
+
+  // Check whether we run out of quota
+  const outOfQuota: boolean = await page
+    .getByText('All regions in the selected cloud provider are temporarily unavailable. Select a')
+    .isVisible();
+  console.log('outOfQuota: ' + outOfQuota);
```
Comment (mmusil) [inline lib/kafka.ts:26]: This log seems a bit redundant. Just a nitpick.

Comment (asanzgom) [reply lib/kafka.ts:26]: Can I resolve this conversation?

Comment (asanzgom) [reply lib/kafka.ts:26]: @mmusil

**Comment by asanzgom** [comment]
Can I merge @mmusil ?

**Pull request merged**